server/command.go

```diff
@@ -399,12 +399,13 @@ func getCommand() *model.Command {
 	}
 }
 
-func responsef(format string, args ...interface{}) *model.CommandResponse {
+func responsef(p *Plugin, format string, args ...interface{}) *model.CommandResponse {
+	conf := p.getConfig()
 	return &model.CommandResponse{
 		ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
 		Text:         fmt.Sprintf(format, args...),
-		Username:     PluginMattermostUsername,
-		IconURL:      PluginIconURL,
+		Username:     conf.UserName,
```

Comment:
Does this mean it's now configurable?

Reply:
Yes, it will use the same configured username wee use for webhook posts. I assumed this was desirable, right?